src/qrl/core/TransactionPool.py

```diff
@@ -15,6 +15,17 @@
 
 
 class TransactionPool:
+    """
+    As each Transaction comes in over the network, it is first stored in
+    TransactionPool.pending_tx_pool before the node validates them. After the TX
+    has been validated, it is moved to TransactionPool.transaction_pool.
+
+    The node pulls from this Pool in order to fill a block, which it then
+    attempts to mine. If a block is successfully mined, it will naturally be
+    added to the Chain, at which point ChainManager will call
+    remove_tx_in_block_from_pool()
```

Comment:
Better to have this comment under remove_tx_in_block_from_pool function, instead of talking here.